streaming/redpanda/README.md

```diff
@@ -0,0 +1,305 @@
+# Redpanda on Amazon EKS
+****
+Redpanda is a simple, powerful, and cost-efficient streaming data platform that is compatible with Kafka APIs but much less complex, faster and more affordable. Get started in seconds with Redpanda Serverless!
+****
+![redpanda.png](redpanda.png)
+## Prerequisites
+
+This guide also has the following prerequisites for setting up and deploying Redpanda in EKS.
+
+* Terraform
+* AWS-CLI
+* Kubectl
+* Helm
+* jq
+
+****
+
+## Terraform
+****
+Terraform is an infrastructure as code tool that enables you to safely and predictably provision and manage infrastructure in any cloud. Install the latest.
+
+https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli
+
+****
+
+## AWS-CLI
+****
+AWS-CLI is a command line tool for managing AWS resources, install the latest version.
+
+https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html
+****
+
+## Kubectl
+
+Kubectl is a command line tool that is used to communicate with the Kubernetes API server. Install the latest kubectl for your platform from here:
+
+https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html
+
+****
+
+## Helm
+
+Helm is used as the Kubernetes package manager for deploying Redpanda. Install the latest version of Helm for your platform from here:
+
+https://helm.sh/docs/intro/install/
+
+****
+
+Provider Versions Tested
+| Provider     | Version |
+| ----------- | ----------- |
+| Terraform      | v1.7.2      |
+| Helm      |    v3.14.0   |
+| Redpanda   | v5.7.22       |
+
+## Setting up the Cluster
+
+The Redpanda deployment documentation was used extensively in the creation of this guide.  You can view the Redpanda guide at the following location:
+
+https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/kubernetes/eks-guide/
+
+In setting up and configuring our Redpanda cluster in AWS, we are using an Amazon EKS cluster as described in the guide linked above.
+
+We will use Terraform to deploy our AWS Resources (VPC, EKS, EKS add-ons) and our Redpanda Cluster, clone the files from Git.
+
+
+To stand up the Amazon EKS Cluster and Infrastrucuture, run Terraform init, plan and apply.
+```
+terraform init
+terraform plan
+terraform apply --autor-approve
+```
+
+After some time, you should see output that includes the message “Congratulations on installing redpanda!”, along with information on your cluster as well as a number of sample commands that you can use.
+
+Create Access to your Amazon EKS Cluster
+```
+aws eks --region us-west-2 update-kubeconfig --name doeks-redpanda
+```
+Verify Cluster
+```
+kubectl get nodes
+```
+You should see 6 worked nodes, 3 deployed for EKS Add-ons (Cert-Manager, AWS Load Balancer Controller, Monitoring Tools, etc).
+
+## Configuring Environment for Lambda/Serverless Lab
```

Comment:
Please remove Lambda references since is not anymore used.

Reply:
Complete